src/main/java/frc/robot/commands/intake/armIntake/IntakeCube.java

```diff
@@ -23,7 +31,7 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    intake.coneDrop(Constants.INTAKE_SPEED);
+    intake.coneDrop(speed);
```

Comment:
This should call cubePickup method to be more clear. Must have missed this in our prior reviews since it’s functionally correct

Reply:
Going to merge this so we can get it to main. This can wait